src/common/selectors/clusterSelectors.ts

```diff
@@ -68,3 +74,12 @@ export const isIPv4 = ({
   allSubnetsIPv4(machineNetworks) &&
   allSubnetsIPv4(clusterNetworks) &&
   allSubnetsIPv4(serviceNetworks);
+
+export const getOlmOperators = (monitoredOperators: MonitoredOperatorsList | undefined) => {
```

Comment:
```suggestion
export const getOlmOperators = (cluster: Partial<Cluster>) => {
```

Reply:
Maybe even `Pick<Cluster, ...>` instead of `Partial<>`? So that we are more explicit about what we are using from the Cluster object in the function?

Reply:
Agree, see the last commits for the update :+1: